app/directives/megasearch.js

```diff
@@ -11,6 +11,14 @@ angular
 			},
 			link: function(scope, elem) {
 				scope.provideSuggestions = function(keyCode) {
+
+					// size the mega search box to match the input field
+					if($location.path()==='/'){
+						$('#search-terms #megasearchbox').width($('#search-terms #searchbox').width()-18);
```

Comment:
'$' is the jQuery way of selecting things, not angular. this is currently working because jQuery loads before this is ran, but we cannot rely on jQuery being there. Please modify this to be angular code.